### PR TITLE
Handle instance tags and instance cpu affinity

### PR DESCRIPTION
**Pull request by mandoonandy**
Depends on shakenfist/shakenfist#1134

### CODE REVIEW

**Comment by mikalstill** [comment]
This is out of date compared with what landed in the server isn't it? Shouldn't this be more generic metadata now?

File: shakenfist_client/commandline/instance.py [hunk 1]
```diff
@@ -408,14 +422,27 @@ def instance_create(ctx, name=None, cpus=None, memory=None, network=None, floate
                 return
             video[s[0]] = s[1]
 
+    metadata_def = {}
+    for m in metadata:
+        try:
+            key, val = m.split('=')
+        except ValueError:
+            print('Unable split metadata, correct format is key=value')
```
Comment (mikalstill) [inline shakenfist_client/commandline/instance.py:430]: "Unable to"? I wouldn't nit, except its user facing.

**Review by mikalstill — CHANGES_REQUESTED** [review]
LGTM, apart from nit.

File: shakenfist_client/commandline/instance.py (hunk 1 above)
Comment (mandoonandy) [reply shakenfist_client/commandline/instance.py:430]: Good pick up.

**Pull request merged**